content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -138,7 +114,7 @@ following commands on a machine with cluster admin privileges:
 
     {{< text bash >}}
     $ go run istio.io/istio/security/tools/generate_cert \
-          -client -host spiffee://cluster.local/vm/vmname --out-priv key.pem --out-cert cert-chain.pem  -mode citadel
+          -client -server -host spiffe://cluster.local/ns/vm/sa/default  --out-priv key.pem --out-cert cert-chain.pem  -mode citadel
```

Comment:
@irisdingbj this command will work only if istiod is running self-signed mode, right?